src/onboarding/dialogs/app-dialog.ts

```diff
@@ -0,0 +1,686 @@
+import { LitElement, css, html, nothing } from "lit";
+import { customElement, property } from "lit/decorators";
+import { LocalizeFunc } from "../../common/translations/localize";
+import { fireEvent } from "../../common/dom/fire_event";
+import { createCloseHeading } from "../../components/ha-dialog";
+
+@customElement("app-dialog")
+class DialogApp extends LitElement {
+  @property({ attribute: false }) public localize?: LocalizeFunc;
+
+  public async showDialog(params): Promise<void> {
+    this.localize = params.localize;
+  }
+
+  public async closeDialog(): Promise<void> {
+    this.localize = undefined;
+    fireEvent(this, "dialog-closed", { dialog: this.localName });
+  }
+
+  protected render() {
+    if (!this.localize) {
+      return nothing;
+    }
+    return html`<ha-dialog
+      open
+      @closed=${this.closeDialog}
+      .heading=${createCloseHeading(
+        undefined,
+        this.localize("ui.panel.page-onboarding.welcome.download_app")
+      )}
+    >
+      <div>
+        <a
+          href="https://apps.apple.com/us/app/home-assistant/id1099568401"
+          target="_blank"
+          rel="noreferrer noopener"
+        >
+          <svg viewBox="0 0 160 227" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Should we move the svg into their own files?

Reply:
I don't think that makes sense? It will just cause extra small requests?